### ISSUE TITLE
Establish the relationship between git repo and GSA System

### ISSUE THREAD

**Issue by JJediny** [18F/tts-tech-portfolio#1179]
The Tech Portfolio needs the ability to quickly associate the relationships between git repositories/codebase(s) to their system owners.

@its-a-lisa-work as lead developed https://app.smartsheet.com/sheets/4jVMQjG7mw3H764hx2PmWhXvjCm5fjv5cMwVcWX1?view=grid

## Implementation 
- [ ] Determine the best means to source these relationships

## Acceptance Criteria
- [ ] A process for establishing the link between github repos and their system owners is documented and communicated

**Comment by its-a-lisa-at-work** [comment]
The relationships that are needed are
Repo -> Website
Repo -> (FISMA) System
Website - > System

Currently, I put together a sheet that has this mapping -- but unless there is a process, person or technology responsible with keeping it up to date, it is going to go out of date quickly. It took about 50hrs for me to gather to get a high level degree of certainty that it is accurate. There was a lot of manual labor involved because I had to physically(ish) go to over 300 websites and determine if they 'existed' -- even if they did exist, there is no telling whether or not this is a `production` or `staging` site. And for some sites that don't exist, I do not feel confident that there isn't something there based on this just being a visual inspection. The method that I used to get the list is also flawed since I only had the two lists to rely on and it wasn't until I received the Federalist list that new sites emerged that were previously unknown to both GSA and the TTS Tech Portfolio; some that aren't even on our radar. 

The solution relates to asset inventory and vulnerability management. 

A suggested solution is to have a tool such as [Dectify](https://detectify.com/product/asset-monitoring) help us with asset monitoring.
